_automating-configurations/security.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Workflow security
+nav_order: 25
+---
+
+# Flow Framework security
```

Comment:
Kept it as security to stay consistent with other plugins https://opensearch.org/docs/latest/observing-your-data/ad/security/. Let me know if you think otherwise